core/templates/dev/head/pages/dashboard/dashboard.html

```diff
@@ -194,79 +196,133 @@ <h1 class="stat-value"><[dashboardStats.total_open_feedback]></h1>
       </div>
 
       <div ng-if="myExplorationsView === 'card' && explorationsList.length > 0"
-           style="margin-top: 8px;">
-        <md-card ng-repeat="exploration in explorationsList track by exploration.id"
+           class="oppia-card-view-wrap">
+        <md-card ng-repeat="exploration in explorationsList | orderBy:sortByFunction:isCurrentSortDescending track by exploration.id"
                  class="oppia-activity-summary-tile oppia-dashboard-card-view-item"
-                 ng-click="showExplorationEditor(exploration.id)">
-            <div class="title-section" style="background-color: <[exploration.thumbnail_bg_color]>;">
-              <img class="thumbnail-image" ng-src="<[exploration.thumbnail_icon_url]>">
-              <h2 class="activity-title protractor-test-exp-summary-tile-title">
-                <span><[exploration.title || 'Untitled' | truncate:40]></span>
-              </h2>
-            </div>
+                 ng-class="{'dropdown-open': activeExplorationId === exploration.id}"
+                 scroll-on-click
+                 is-dropdown-open="activeExplorationId === exploration.id"
+                 is-private="exploration.status === 'private'"
+                 ng-click="openExploration(exploration.status, exploration.id)">
+          <div class="title-section"
+               ng-class="{'oppia-exp-arrow': activeExplorationId === exploration.id}"
+               style="background-color: <[exploration.thumbnail_bg_color]>;">
+            <img class="thumbnail-image"
+                 ng-src="<[exploration.thumbnail_icon_url]>"
+                 ng-hide="activeExplorationIdOnMobile(exploration.id)">
+            <span class="thumbnail-image caret-thumb fa fa-caret-up"
+                  ng-show="activeExplorationIdOnMobile(exploration.id)">
+            </span>
+            <h2 class="activity-title protractor-test-exp-summary-tile-title">
+              <span><[exploration.title || 'Untitled' | truncate:40]></span>
+            </h2>
+          </div>
+          <div class="mask-wrap">
             <div class="title-section-mask"></div>
-
-            <div ng-attr-section="'<['right-section']>">
-              <div ng-if="exploration.status === 'private'" class="exp-private-text">
-                <[publishText]>
+          </div>
+          <md-card class="oppia-exp-dropdown-stats"
+                   ng-click="$event.stopPropagation()"
+                   ng-if="exploration.status !=='private' && activeExplorationId === exploration.id">
+            <div class="stat-block">
+              <div class="heading-text"
+                   ng-click="toggleSubDropdown(EXPLORATION_DROPDOWN_STATS.OPEN_FEEDBACK, $event)">
+                <[explorationStats[exploration.id] ? explorationStats[exploration.id].open_feedback.length + ' Open Feedback': 'Retrieving Feedback Data ...']>
```

Comment:
It seems like this is causing the dashboard to display "3 Open Feedback" even when there are more than 3. If there are more than 3, you should probably use the feedback count used at the top of the dashboard page, and also indicate that the feedback is truncated (perhaps with "and x more" at the bottom).